src/OmniSharp.DotNetTest/Models/RunTestInClassRequest.cs

```diff
@@ -0,0 +1,16 @@
+﻿using OmniSharp.Mef;
+using OmniSharp.Models;
+
+namespace OmniSharp.DotNetTest.Models
+{
+    [OmniSharpEndpoint(OmniSharpEndpoints.V2.RunAllTestsInClass, typeof(RunTestsInClassRequest), typeof(RunTestResponse[]))]
+    public class RunTestsInClassRequest : Request
```

Comment:
I feel like this could share a base class with the DebugTests request (they're identical, no?)